src/platforms/android.ts

```diff
@@ -91,6 +120,26 @@ export class AndroidPlatform extends Platform {
     ];
   }
 
+  private async shortcutIcons(): Promise<FaviconImage[]> {
+    const images = new Images();
+    const icons = await Promise.all(
+      this.options.shortcuts.map(async (shortcut, index) => {
+        if (!shortcut.name || !shortcut.url || !shortcut.icon) return null;
+        const shortcutSourceset = await sourceImages(shortcut.icon);
+        return Promise.all(
+          Object.entries(SHORTCUT_ICONS_OPTIONS).map(([shortcutName, option]) =>
+            images.createFavicon(
+              shortcutSourceset,
+              `shortcut${index + 1}-${shortcutName}`,
+              option
+            )
+          )
+        );
+      })
+    );
+    return icons.flat();
```

Comment:
Just realized that this isn't supported in Node.js 10. Are you happy to require Node.js 12+ in v7 and dropping support for this EOL Node.js version or do I need to rewrite this?

Reply:
That's totally OK.